data/campaigns/Delfadors_Memoirs/scenarios/02_This_Valley_Belongs_to_Me.cfg

```diff
@@ -65,7 +65,7 @@
             story = _ "Delfador’s wandering time with Methor took them to many strange places, both within and beyond the lands of men. They studied and practiced, applying magecraft to aid those they traveled among and to deepen their understanding of the hidden currents and powers of the world."
         [/part]
         [part]
-            story = _ "But the elder mage’s health was taxed by the rigors of the journey. A year after the two left Alduin, Methor settled in a tranquil valley at the edge of the Dulatus Hills, not far southeast of the capitol at Weldyn. He opened a small school of his own, and with Delfador assisting him he began training a handful of youths who might in time be worthy to enter the great Academy on Alduin."
+            story = _ "But the elder mage’s health was taxed by the rigors of the journey. A year after the two left Alduin, Methor settled in a tranquil valley at the edge of the Dulatus Hills, not far southeast of the capitol at Weldyn. He opened a small school of his own; and with Delfador assisting him, he began training a handful of youths who might in time be worthy to enter the great Academy on Alduin."
```

Comment:
@max-torch @nemaara I'm sorry I missed the opportunity to correct this in this big review. With respect to:

> ...the capitol at Weldyn.

As I understand it, there's the US Capitol Building in the US capital, but it's incorrect - though seemingly rather common in US English - to use 'capitol' when 'capital' should actually be used instead, as is the case here. Any objection to changing this once the 1.16.2 string freeze is over?

Reply:
I wouldn't object. @Wedge009